common/metrics/scope.go

```diff
@@ -120,7 +120,7 @@ func (m *metricsScope) RecordHistogramValue(id int, value float64) {
 }
 
 func (m *metricsScope) Tagged(tags ...Tag) Scope {
-	domainTagged := false
+	domainTagged := m.isDomainTagged
 	tagMap := make(map[string]string, len(tags))
```

Comment:
aaah.  yeah, this makes sense - otherwise we'd lose the value when we make a cached scope with a domain tag.  👍